cni/network/multitenancy.go

```diff
@@ -64,18 +64,14 @@ func getContainerNetworkConfiguration(
 	return getContainerNetworkConfigurationInternal(nwCfg.CNSUrl, podNamespace, podNameWithoutSuffix, ifName)
 }
 
-func getContainerNetworkConfigurationInternal(
-	address string,
-	namespace string,
-	podName string,
-	ifName string) (*cniTypesCurr.Result, *cns.GetNetworkContainerResponse, net.IPNet, error) {
+func getContainerNetworkConfigurationInternal(address string, namespace string, podName string, ifName string) (*cniTypesCurr.Result, *cns.GetNetworkContainerResponse, net.IPNet, error) {
 	cnsClient, err := cnsclient.GetCnsClient()
 	if err != nil {
 		log.Printf("Failed to get CNS client. Error: %v", err)
 		return nil, nil, net.IPNet{}, err
 	}
 
-	podInfo := cns.KubernetesPodInfo{PodName: podName, PodNamespace: namespace}
+	podInfo := cns.NewPodInfo("", "", podName, namespace)
```

Comment:
I dont think we can change this, existing partners CNI's are not updated to latest CNI version, so this will break with just cns update